src/org/omegat/filters2/html2/HTMLFilter2.java

```diff
@@ -179,7 +179,7 @@ public void processFile(BufferedReader infile, BufferedWriter outfile,
             parser.setInputHTML(all.toString());
             parser.visitAllNodesWith(new FilterVisitor(this, outfile, options));
         } catch (ParserException pe) {
-            System.out.println(pe);
+            Log.log(pe);
```

Comment:
it would be better to be `Log.logErrorRB(pe, "<bundle key>");

Reply:
Thanks, edited accordingly